### ISSUE TITLE
Do not have module "resource" that is needed after build 61

### ISSUE THREAD

**Issue by suresttexas00** [benbaptist/minecraft-wrapper#51]
The last working build was 61.  This error continues to plague me:
I have tracked this down as starting with build 62...What was done in that build that would cause this?
(started from windows batch file):

C:\Users\JRUSSELL\Desktop\vanilla>python Wrapper.py
Traceback (most recent call last):
  File "C:\Python27\lib\runpy.py", line 162, in _run_module_as_main
    "__main__", fname, loader, pkg_name)
  File "C:\Python27\lib\runpy.py", line 72, in _run_code
    exec code in run_globals
  File "Wrapper.py\__main__.py", line 7, in <module>
  File "Wrapper.py\server.py", line 1, in <module>
ImportError: No module named resource

C:\Users\JRUSSELL\Desktop\vanilla>pause
Press any key to continue . . .

**Comment by suresttexas00** [comment]
I'm pretty sure this is the problem:
+import socket, datetime, time, sys, threading, random, subprocess, os, json, signal, traceback, api, StringIO, ConfigParser, backups, sys, codecs, resource


I dont have that module and dont know where to get it(resource).

**Comment by benbaptist** [comment]
wait, that's weird. I actually forgot what resource is - I wonder if that's
a typo and I just so happen to have it installed, or I'm just really brain
farting. Either way, I'm going to investigate right now.

On Tue, Jan 6, 2015 at 7:17 PM, Surest Texas <notifications@github.com>
wrote:

> I'm pretty sure this is the problem:
> +import socket, datetime, time, sys, threading, random, subprocess, os,
> json, signal, traceback, api, StringIO, ConfigParser, backups, sys, codecs,
> resource
>
> I dont have that module and dont know where to get it(resource).
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/benbaptist/minecraft-wrapper/issues/51#issuecomment-68958871>
> .
>



-- 
http://sawmaster2.no-ip.org:82/

iPod network

**Comment by benbaptist** [comment]
Aha! I remember now. I used it to aid in getting server memory usage. It
doesn't work on Windows, and it doesn't have a Windows equivalent, so for
now it'll just be disabled if you're not using a POSIX system. Pushing code
in a few moments!


On Tue, Jan 6, 2015 at 11:09 PM, Ben Baptist <sawham6@gmail.com> wrote:

> wait, that's weird. I actually forgot what resource is - I wonder if
> that's a typo and I just so happen to have it installed, or I'm just really
> brain farting. Either way, I'm going to investigate right now.
>
> On Tue, Jan 6, 2015 at 7:17 PM, Surest Texas <notifications@github.com>
> wrote:
>
>> I'm pretty sure this is the problem:
>> +import socket, datetime, time, sys, threading, random, subprocess, os,
>> json, signal, traceback, api, StringIO, ConfigParser, backups, sys, codecs,
>> resource
>>
>> I dont have that module and dont know where to get it(resource).
>>
>> --
>> Reply to this email directly or view it on GitHub
>> <https://github.com/benbaptist/minecraft-wrapper/issues/51#issuecomment-68958871>
>> .
>>
>
>
>
> --
> http://sawmaster2.no-ip.org:82/
>
> iPod network
>



-- 
http://sawmaster2.no-ip.org:82/

iPod network

**Comment by suresttexas00** [comment]
well, that explains why I was able to simply delete the reference to get everything working again!

**Comment by suresttexas00** [comment]
removing the reference worked and I have been using build 69 successfully.  I just started using this again.   I was using another wrapper, but I was limited to ! or .<command> in chat.  Which I was willing to live with... until it started having problems with server crashes and not restarting.  After coming back to this, I actually find the plugins are much easier to write on this platform.  I'm going to close all the other issues I opened, as everything is working fine now.
